### ISSUE TITLE
download_test_file() fails

### ISSUE THREAD

**Issue by t-sommer** [CATIA-Systems/FMPy#53]
The following code fails because all requests to https://trac.fmi-standard.org/ are now redirected to https://github.com/modelica/fmi-standard/issues/.

```
from fmpy.util import download_test_file

download_test_file('FMI_2.0', 'CoSimulation', 'Dymola', '2017', 'Rectifier', 'Rectifier.fmu')
```

This is also breaking [some tests](https://travis-ci.org/CATIA-Systems/FMPy/jobs/445539485#L1142).

**Comment by chrbertsch** [comment]
You can still acces the the test FMUs via https://svn.modelica.org/fmi/branches/public/Test_FMUs/

**Comment by t-sommer** [comment]
Fixed with ef340d7ff7bad4bdc1689d293edffeb2d87d9bc9
